app/views/kalibro_configurations/index.html.erb

```diff
@@ -1,16 +1,16 @@
 <div class="page-header">
-  <h1>Configurations</h1>
+  <h1><%= KalibroConfiguration.model_name.human(count: 2) %></h1>
 </div>
 
 <% if user_signed_in? %>
   <p>
-    <%= link_to 'New Configuration', new_kalibro_configuration_path, class: 'btn  btn-primary' %>
+    <%= link_to "#{t('new.female')} #{KalibroConfiguration.model_name.human}", new_kalibro_configuration_path, class: 'btn  btn-primary' %>
```

Comment:
We are going to discuss this soon. I do agree with you but fixing this will be a history of its own.